src/wmtk/Scheduler.cpp

```diff
@@ -17,6 +17,8 @@ void Scheduler::run_operation_on_all(PrimitiveType type, const std::string& name
     // run();
```

Comment:
success/fail need to be reset here in the case that the scheduler gets called multiple times (this is the main reason we want this - to call teh same scheduler multiple times until all operations fail). A unit test that catches this should exist too